Rocket.Chat/Controllers/Chat/ChatViewController.swift

```diff
@@ -361,12 +361,18 @@ final class ChatViewController: SLKTextViewController {
                     message.map(response.result["result"], realm: realm)
                     realm.add(message, update: true)
 
-                    MessageTextCacheManager.shared.update(for: message)
+                    MessageTextCacheManager.shared.update(for: message, asyncUpdate: nil)
                 })
             }
         }
     }
 
+    fileprivate func updateCellForMessage(identifier: String) {
+        guard let indexPath = self.dataController.indexPathOfMessage(identifier: identifier) else { return }
+
+        self.collectionView?.reloadItems(at: [indexPath])
```

Comment:
What do you think about adding it inside of a `UIView.performWithoutAnimation()` block?